gradle/release.gradle

```diff
@@ -11,6 +11,9 @@ releasing {
     git.releasableBranchRegex = "master|release/.+" // matches 'master', 'release/2.x', 'release/3.x', etc.
     git.branch = System.getenv("TRAVIS_BRANCH")
 
+    def buildNo = System.getenv("TRAVIS_BUILD_NUMBER")
```

Comment:
just saw your recent changes in mockito-release-tools (TravisPlugin and so on). I really like the new TravisPlugin and therefore get rid of this configuration :+1: